src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -352,6 +325,36 @@ def _storeFieldMetaInfo(self, node_kw: Union[dict, list]):
         self._config_node_meta[name].output_file = out_file
         self._config_node_meta[name].var_type = var_type
 
+    @staticmethod
+    def gen_data_node(gen_data: List[str]) -> Optional[GenDataConfig]:
+        options = _option_dict(gen_data, 1)
+        name = gen_data[0]
+        res_file = options.get(ConfigKeys.RESULT_FILE)
+
+        if res_file is None:
+            raise ConfigValidationError(
+                f"Missing or unsupported RESULT_FILE for GEN_DATA key {name!r}"
+            )
+
+        report_steps = rangestring_to_list(options.get(ConfigKeys.REPORT_STEPS, ""))
+
+        if os.path.isabs(res_file) or "%d" not in res_file:
+            msg = (
```

Comment:
Nitpick, I would prefer `logger.error(
                "The GEN_DATA keywords must have a REPORT_STEPS:xxxx defined"
                "Several report steps separated with ',' and ranges with '-'"
                "can be listed"
            )`

Reply:
Yep! :+1: